Directory.Packages.props

```diff
@@ -124,7 +124,7 @@
     <PackageVersion Include="Microsoft.Signed.Wix" Version="1.0.0-v3.14.0.5722" />
     <PackageVersion Include="Microsoft.DotNet.Build.Tasks.Installers" Version="8.0.0-beta.23564.4" />
     <!-- unit test dependencies -->
-    <PackageVersion Include="Microsoft.Extensions.Diagnostics.Testing" Version="8.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.Diagnostics.Testing" Version="8.1.0" />
```

Comment:
This comes from dotnet/extensions, right? Maybe it should be on the same update plan as Http.Resilience.

Reply:
Oh, great catch, yeah this should be using dependency flow, I'll fix this.